info.mumble.Mumble.appdata.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It would be preferred to get it merged upstream first to make every distribution benefit.